pkgs/love2d/example/assets/modern_pong/main.lua

```
@@ -0,0 +1,18 @@
+GameStateManager = require("libs/gamestateManager")
+local MainMenu = require("states/mainMenu")
+
+function love.load()
+    GameStateManager:setState(MainMenu)
+end
+
+function love.update(dt)   
```

Luacheck (1.2.0)

[warning] line contains trailing whitespace

(W612)